playbooks/ceph.yml

```diff
@@ -63,7 +63,7 @@
   become: true
   vars:
     cifmw_admin_user: ceph-admin
-    ansible_ssh_private_key_file: "{{ lookup('env', 'ANSIBLE_SSH_PRIVATE_KEY') }}"
+    ansible_ssh_private_key_file: "{{ hostvars[(groups[cifmw_ceph_target | default('computes')] | default([]) | first)]['ansible_ssh_private_key_file'] | default(lookup('env', 'ANSIBLE_SSH_PRIVATE_KEY')) }}"
```

Comment:
I assume this preserves the default behavior as the default value is what we previously had on L66.

Reply:
yes